### ISSUE TITLE
[feat] Start new project from Clipboard Content

### ISSUE THREAD

**Issue by alolalo** [phcode-dev/phoenix#1011]
### Describe the problem

I just want to copy-paste some code into an editor for later processing, not setting up a boilerplate project

### Describe the solution you'd like

The content of the clipboard is auto-detected and what language it uses, but would be elementary support HTML, CSS & JS.

Once I paste the code it goes into a seperate "Inbox" folder in which these "Snippets" are stored. The inbox can be shared across multiple projects so it would function as a backbone building block, not a front-facing building block like UI-components.

### Alternatives considered

Whut? You pit yourselfs against competition when you're an online IDE that saves me from offline alternatives!

### Additional context

I'm a designer who devs like a Dutch Hell-diver of Details called justgu.us
Questions? Just askgu.us

**Comment by alolalo** [comment]
Love the experience, hope you can make some UX changes like bigger typography so I can read the code a bit better

**Comment by abose** [comment]
Thanks for the feature request @alolalo . We need to have a better-integrated snippets manager as you suggested. But in the meantime, can you try out the extension `Brackets Snippets (by edc)` if it suits your needs?

![image](https://user-images.githubusercontent.com/5336369/226825425-6a2378c8-d04a-4ea4-b833-dc0df66768e6.png)

I have created an issues to track the feature request: https://github.com/phcode-dev/phoenix/issues/1014

Please provide any feedback on how you would ideally want the workflow or the problems faced in detail to help more in understanding what needs to be done.

**Comment by abose** [comment]
Issue tracking:  bigger typography so I can read the code a bit better
https://github.com/phcode-dev/phoenix/issues/1015

**Comment by alolalo** [comment]
Yes this helps a bit I guess, and I do agree that the typography needs to get more user-controlled because it's kind of unbearable to read, in app too. I hope that helps you people improve some of the basics that I think people are used to.
